app/views/nonprofits/_actions.html.erb

```diff
@@ -5,18 +5,18 @@
 <aside class='u-marginBottom--15'>
   <% if current_nonprofit_user? %>
 	  <button class='button--jumbo  u-width--full' open-modal='newCampaign'  if-branded='background-color, dark'>
-			Start a Campaign
+			<%= t('nonprofits.actions.start_campaign') %>
     </button>
   <% else %>
     <a class='button u-width--full' if-branded='background-color, dark' target='_blank' href='/peer-to-peer?npo_id=<%=@nonprofit.id %>'>
-      Start a Campaign for <%= @nonprofit.name %>
+      <%= t('nonprofits.actions.start_campaign_for') %> <%= @nonprofit.name %>
```

Comment:
One thing that's important about i18n is making sure you use the right abstraction. For example here, in English, the text is `Start a Campaign For Nonprofit Name`. But in other languages, nonprofit might go first or in a different place in the string. The information on proper abstraction should describe that in https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations.

Also, please change this name to  `nonprofits.page.actions.start_campaign_for` and use `nonprofit_name` as the name of the argument.